### PR TITLE
Summarize failed tests

### PR DESCRIPTION
**Pull request by khanh-nguyen**
A little summary at the end can be useful when the log is long.

### CODE REVIEW

File: bin/tap-difflet [hunk 1]
```diff
@@ -154,4 +158,13 @@ process.on('exit', function () {
   process.exit(errors.length);
 });
 
+function summarizeErrors(errors) {
+  var summary = chalk.underline.bold('\n\nTests that failed:\n');
+
+  _.each(errors, function(error) {
```
Comment (NickHeiner) [inline bin/tap-difflet]: You could use `errors.forEach` and not need to add lodash as a dep, unless we're concerned about supporting really old versions of Node.

**Comment by NickHeiner** [comment]
@khanh-nguyen can you include a picture or console output copy/paste in the PR description of what this change looks like?

File: bin/tap-difflet (hunk 1 above)
Comment (khanh-nguyen) [inline bin/tap-difflet]: :+1: You're right. It's a bit overkill as well.

**Comment by khanh-nguyen** [comment]
@NickHeiner  my bad, I didn't see this message until now. Here's a screenshot of how the result looks like. 

screen shot 2016-03-17 at 1 40 33 am

**Comment by namuol** [comment]
Closing since the recently-added `-p` flag (0.5.0) helps remove noise.

**Pull request closed without merging**